ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -418,10 +421,13 @@ applyByronGenTx :: Bool -- ^ Have we verified this transaction previously?
 applyByronGenTx _reapply (ByronLedgerConfig cfg) genTx st@ByronLedgerState{..} =
     (\x -> st { blsCurrent = x }) <$> go genTx blsCurrent
   where
+    validationMode = fromBlockValidationMode CC.Block.BlockValidation
```

Comment:
For now, I'm just hardcoding the `BlockValidationMode` as `BlockValidation` but I will utilize the `reapply` flag in the PR for #440.

Something like `fromBlockValidationMode (if reapply then CC.Block.BlockValidation else CC.Block.NoBlockValidation)`.